StenosisMeasurement3D/Logic/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+project(vtkSlicer${MODULE_NAME}ModuleLogic)
+
+set(KIT ${PROJECT_NAME})
+
+set(${KIT}_EXPORT_DIRECTIVE "VTK_SLICER_${MODULE_NAME_UPPER}_MODULE_LOGIC_EXPORT")
+
+set(${KIT}_INCLUDE_DIRECTORIES
+  ${ExtraMarkups_ModuleMRML_INCLUDE_DIRS}
+  )
+
+set(${KIT}_SRCS
+  vtkSlicer${MODULE_NAME}Logic.cxx
+  vtkSlicer${MODULE_NAME}Logic.h
+  )
+
+set(${KIT}_TARGET_LIBRARIES
+  ${ITK_LIBRARIES}
```

Comment:
Need to add here `vtkSlicerShapeModuleMRML`. Without that build fails on Windows due to unresolved symbol.

Reply:
I'll need some clarification to fix the dependency issue.

The reasonable step seems adding

`set(EXTENSION_DEPENDS "ExtraMarkups")`

in the CMakeLists.txt file of SlicerVMTK.

It's the modification in the s4ext file of ExtraMarkups that is not clear.

The other modifications are quite straightforward.

Thank you for considering.

Reply:
Update your CMakeLists.txt file and then build your project. It will generate an updated s4ext file that contain the new dependency.

Reply:
> It will generate an updated s4ext file that contain the new dependency.

Ok, what I just understood is that, in parallel, it's up to me to submit the updated s4ext file of SlicerVMTK to the 'Extension index' repository. I thought it was a transparent process as SlicerVMTK is already there since long. I suppose that it should be done for the master and 5.2 branches of the 'Extension index'.

I'll do these ASAP.

Reply:
The extension index in theory could look up metadata fields from the repository, but currently it requires the repository url, tag, and dependencies to be up-to-date in the s4ext file.

It is better to add the dependency to the s4ext file before the code in your repository starts to use the dependency.